reporters_db/__init__.py

```diff
@@ -34,6 +35,11 @@ def datetime_parser(dct):
     CASE_NAME_ABBREVIATIONS = json.load(f)
 
 
+with open(os.path.join(db_root, "data", "variables.json")) as f:
+    RAW_VARIABLES = json.load(f)
+    VARIABLES = process_variables(RAW_VARIABLES)
```

Comment:
Just calling these `VARIABLES` seemed a bit too generic to me. Maybe call them `REGEX_VARIABLES` instead?